### DISCUSSION TITLE
How to ignore git submodule upgrade without disabling future PRs

### DISCUSSION THREAD

**Discussion by tennox** [renovatebot/renovate#13092] [Request Help] [answered]
I'm tracking the version of a dependency [via git submodule](https://gitlab.com/txlab/docker-linkcheck), and some master commits I would like to ignore. But when I closed the PR, [this message was sent by renovate](https://gitlab.com/txlab/docker-linkcheck/-/merge_requests/3#note_755771558):

> As this MR has been closed unmerged, Renovate will ignore this upgrade updateType and you will not receive MRs for any future linkcheck:master digest updates. Digest updates will resume if you update the specified tag at any time.

But I do want to receive MRs for future linkched:master commits. Is it possible?

**Comment by rarkins** [comment, marked as answer]
You'll get a new PR if either the branch name or the PR title changes. If the PR title included a SHA in it, then that should mean any future changes trigger PRs.

**Reply by tennox** [reply]
The PR title is `Update linkcheck commit hash to 8b74603` so it should create new PR for a new commit. :tada: 

If that's the case I'd argue the comment by renovate is a bit misleading:
> you will not receive MRs for **any future linkcheck:master digest updates**

**Reply by rarkins** [reply]
The relevant string is here: https://github.com/renovatebot/renovate/blob/46930ddbc97a24a0285a0c32033589c3e7b935e0/lib/config/options/index.ts#L2203-L2204

You are welcome to PR an improvement

**Reply by tennox** [reply]
[:heavy_check_mark: submitted PR](https://github.com/renovatebot/renovate/pull/13108)
